Remember The Date/AppDelegate.m

```diff
@@ -70,24 +62,14 @@ -(void) setupSDKStyle {
     if ([ZDKUIUtil isNewerVersion:@6]){
         [[UINavigationBar appearance] setTintColor:[UIColor whiteColor]];
         [[UINavigationBar appearance] setBarTintColor:RED_COLOR];
-        [[ZDKUITextView appearance] setTintColor:RED_COLOR];
         [createRequestSpinner setTintColor:RED_COLOR];
         [requestListSpinner setTintColor:RED_COLOR];
         [commentListSpinner setTintColor:RED_COLOR];
     } else {
         [[UINavigationBar appearance] setTintColor:RED_COLOR];
     }
-    
-    [[ZDKCreateRequestView appearance] setSpinner:(id<ZDKSpinnerDelegate>)createRequestSpinner];
-    [[ZDRequestListLoadingTableCell appearance] setSpinner:(id<ZDKSpinnerDelegate>)requestListSpinner];
```

Comment:
We can go ahead and remove the declaration and modification of these spinners too.